.travis.yml

```diff
@@ -35,6 +36,14 @@ script:
       pdflatex std;
     fi
   - popd
+  # Fail if there is whitespace at the ends of any lines
+  - export TEST_STATUS=0;
+    if [ "$BUILD_TYPE" = "check-whitespace" ]; then
+      grep '\s$' source/*.tex && export TEST_STATUS=1;
```

Comment:
Hm, I don't know Travis, but if this is just shell code, couldn't you just say `grep '\s$' source/*.tex` here and let `grep`'s return status do the job? It seems that just `false` has the desired effect of failing the test, so `grep` should be able to d the same?